packages/js/data/src/payment-settings/types.ts

```diff
@@ -73,39 +74,52 @@ export type PaymentProviderOnboardingState = {
 	completed: boolean;
 	test_mode: boolean;
 	wpcom_has_working_connection?: boolean;
+	wpcom_is_store_connected?: boolean;
+	wpcom_has_connected_owner?: boolean;
+	wpcom_is_connection_owner?: boolean;
 };
 
-// General payment provider type.
-export type PaymentProvider = {
+// Represents a payments entity, which can be a payment provider or a suggested payment extension outside providers.
+export type PaymentsEntity = {
```

Comment:
This is the most general type that includes both providers in the main providers list and suggestions from the Others section.